packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -283,7 +359,9 @@ function build(
       }
       parseVatSlot(slot); // assertion
       valToSlot.set(val, slot);
-      slotToVal.set(slot, val);
+      slotToVal.set(slot, new WeakRef(val));
```

Comment:
Huh, good point. I think in this case we don't register it (because it's an export), but that warrants a thorough review, and it looks like I'm not removing things from the `deadSet` at the right times. I'll take another pass.